File: apps/mobile_flutter/lib/core/config/build_info.dart
/// Shown in the app's top bar so you can tell, after a restart + hard
/// refresh, whether the browser actually picked up the latest build.
///
/// Bump [version] and [updatedAt] on every code change that touches the
/// Flutter app, per the workflow in CLAUDE.md. If the badge in the UI still
/// shows the old version/time after a hard refresh, the frontend has not
/// rebuilt yet -- rerun `Start BhashaVaani.cmd` (which now does a full
/// `flutter build web --wasm`) rather than refreshing again.
class BuildInfo {
  const BuildInfo._();

  static const String version = '0.6.3';
  static const String updatedAt = '2026-08-02 07:00 IST';
  static const String summary = 'Fixed "Completed" count being cross-language and duplicate-inflated (progress now scoped to selected language); roadmap tab statuses updated';
}

File: apps/web_pwa/src/core/config/buildInfo.ts
// Mirrors apps/mobile_flutter/lib/core/config/build_info.dart.
//
// Shown in the app's top bar so you can tell, after a rebuild + reload,
// whether the browser actually picked up the latest build. Bump `version`
// and `updatedAt` on every code change that touches this app, per the
// workflow in CLAUDE.md (same reasoning as the Flutter build badge: Vite's
// dev server hot-reloads fine, but a `vite build` + static-serve deployment
// can still be masked by browser caching, and this badge makes that
// visible instead of guessing from UI behavior).
export const BuildInfo = {
  version: '0.5.1',
  updatedAt: '2026-08-02 07:00 IST',
  summary: 'Fixed "Completed" count being cross-language and duplicate-inflated, which was making Kannada (or other languages) show no phrases left -- Progress tab now scopes to the selected language',
} as const;
